lib/full-site-editing/class-gutenberg-rest-templates-controller.php

```diff
@@ -9,7 +9,7 @@
 /**
  * Base Templates REST API Controller.
  */
-class WP_REST_Templates_Controller extends WP_REST_Controller {
+class Gutenberg_REST_Templates_Controller extends WP_REST_Controller {
```

Comment:
Gutenberg plugin has more logic than core to support file templates... so it needs to use an alternative endpoint.